selfservice/strategy/totp/strategy.go

```diff
@@ -93,7 +93,7 @@ func (s *Strategy) CountActiveMultiFactorCredentials(cc map[identity.Credentials
 			}
 
 			_, err := otp.NewKeyFromURL(conf.TOTPURL)
-			if len(c.Identifiers) > 0 && len(c.Identifiers[0]) > 0 && len(conf.TOTPURL) > 0 && err == nil {
+			if len(conf.TOTPURL) > 0 && err == nil {
```

Comment:
The identifier is not needed for this credential to be valid, as we use the session's identity ID to look up who the user is.